appveyor-ubuntu.yml

```diff
@@ -25,35 +25,21 @@ environment:
   AWS_S3: 'AWS_S3_TESTING'
   AWS_ECR: 'AWS_ECR_TESTING'
   CARGO_LAMBDA_VERSION: "v0.17.1"
- 
+  PYTHON_ARCH: '64'
+  NOSE_PARAMETERIZED_NO_WARN: 1
+  APPVEYOR_CONSOLE_DISABLE_PTY: false
+  APPVEYOR_DETAILED_SHELL_LOGGING: true
+
   matrix:
 
     - PYTHON_HOME: "$HOME/venv3.7/bin"
       PYTHON_VERSION: '3.7'
-      PYTHON_ARCH: '64'
-      NOSE_PARAMETERIZED_NO_WARN: 1
-      INSTALL_PY_38_PIP: 1
```

Comment:
Does it mean that we don't need to specify to install pip for different Python version anymore? Will it install pip automatically?

Reply:
Please check the description above 🙂 

We used to manage python installation and installing the pip before. But we are now using the python versions provided by AppVeyor. 

> Removed following ones as they are not used anymore (they were used once we were managing python versions there, but we are now using the ones which are provided by AppVeyor);
> ```
> INSTALL_PY_38_PIP: 1
> INSTALL_PY_39_PIP: 1
> INSTALL_PY_310_PIP: 1 ```